less/home-page.less

```diff
@@ -1 +1,111 @@
-// Home Content
\ No newline at end of file
+// Home Content
+.home {
+
+    .top-image {
+        width: 100%;
+        margin-bottom: 20px;
+        img {
+            width: 100%;
+        }
+    }//top-image
+
+    .main-content {
+        padding: 0 15px 0 15px;
+        .flex-and-wrap;
+        border-color: @main-content-border-color;
+        
+        .text-content {
+            width: 100%;
+            h1 {
+                font-size: 3.5rem;
+                color: @header-font-color;
+                margin-bottom: 15px;
+            }
+            p {
+                font-size: 1.4rem;
+                line-height: 2rem;
+                color: @paragraph-font-color;
+                margin-bottom: 20px;
+            }
+        }
+        .img-group {
+            display: flex;
+            justify-content: space-between;
+            width: 100%;
+            margin-bottom: 20px;
+            .middle-image {
+                width: 31%;
+                @media @phone {
+                    width: 100%;
+                    margin-left: 20px;
+                    margin-right: 20px;
+                }
+            }
+            img {
+                border-radius: 10px;
+                width: 100%;
+            }
+            @media @phone {
+                .img-sky {
+                    display: none
+                }
+                .img-launch {
+                    display: none;
+                }
+            }
+        }
+    }//main-content
+    
+    .cta {
+        display: flex;
+        justify-content: center;
+        flex-wrap: wrap;
+        border-top: 1px solid @cta-border-color;
+        border-bottom: 1px solid @cta-border-color;
+        padding: 40px 35px 40px 35px;
```

Comment:
T R B L or TB LR, since you set the same for the vertical and horizontal, you can use the shorthand `padding: 40px 35px;`.